imports/ui/Login.tsx

```diff
@@ -0,0 +1,52 @@
+import { Meteor } from "meteor/meteor";
+import React, { useState } from "react";
+import "./Login.css";
+// import faithLogo from "./images/";
```

Comment:
@NathanForgille Go ahead and remove commented out code, unless you are specifically leaving it in there for a reason.